src/Helper.php

```diff
@@ -2,23 +2,19 @@
 
 namespace DeployHuman\fortnox;
 
-
 class Helper
 {
-
-
     /**
-     * Returns a random string of a specified byte size
+     * Returns a random string of a specified byte size,
+     * note that 1 byte contains 2 characters.
      * 
      * @param int $length in bytes
      * @return string $key
      */
     public static function getRandomKey(int $length): string
     {
-        if ($length < 1) {
```

Comment:
it was like this before ;) but sure

Reply:
It was my mistake there, mine was less than `0` for some reason even though the source is less than `1`. Must've changed it when I removed the redundant `$key` variable. 🤷‍♂️

At least the test case supports that `getRandomKey` reverts to a default key length. 😂

Reply:
haha this is why OpenSource is good, more then one pair of eyes! :) your edit is a nice refactor to skip the variable thou! like it

Reply:
Changes the PR title since the original intent didn't really match now. 👍